install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2964,6 +2964,9 @@ sctp:
 
 # Configuration for types of authentication for Cilium
 authentication:
+  # -- Enable authentication processing and garbage collection.
+  # Note that this doesn't affect auth policy enforcement.
```

Comment:
I think this could use a little more explanation - maybe something about what happens when it's disabled? That is, what stops running, and what won't work?

As written, it sounds like disabling authentication won't affect that auth policy enforcement works properly, which doesn't seem right?

Reply:
@youngnick good point. i tried to come up with a better comment. PTAL